### DISCUSSION TITLE
Configuration Options

### DISCUSSION THREAD

**Discussion by Infonautic** [textbrowser/biblioteq#273] [Q&A] [answered]
Hello,

after every Start of BibliotheQ I have to reconnect to the SQL-Lite Database.
I wonder if the is a parameter like "database_path" for the configfile?

Are there other useful but still undocumented parameter?

Thank you.
Thomas

**Comment by textbrowser** [comment]
Yes, there is a terminal option. Is its description not included in the official document?

**Comment by Infonautic** [comment]
Sorry, can't find the option in the documentation.
A Line in the .INI-File would be nice.

**Reply by textbrowser** [reply]
There is Files -> Recent SQLite Files.

**Reply by Infonautic** [reply]
Right, is it possible to open the right database automatically with the start?

**Reply by textbrowser** [reply]
> Right, is it possible to open the right database automatically with the start?

./biblioteq.sh --open-sqlite-database ./Data/BiblioteQ.sqlite 
Launching a local BiblioteQ.

That's in Linux. It should work in Apple too.

**Reply by textbrowser** [reply]
I have not tested it in Apple because it's Apple. Nor Windows.

**Reply by Infonautic** [reply]
I will try it, if it works under Linux I'm get it running on the Mac. No need to open Windows. ;-)
Thanks a lot.

**Reply by textbrowser** [reply]
It's not working in Mac. BQ cannot access the database.

**Reply by textbrowser** [reply]
Correction, it also works in Mac.

open /Applications/BiblioteQ.d/BiblioteQ.app --args --open-sqlite-database absolute-path-to-database-file

**Comment by textbrowser** [comment, marked as answer]
Documented terminal options. You can access an SQLite database via the terminal and the Files menu. Completed.
